pkgs/by-name/pu/purescm/package.nix

```diff
@@ -0,0 +1,45 @@
+{ lib
+, buildNpmPackage
+, fetchNpmDeps
+, testers
+}:
+
+let
+  package = buildNpmPackage rec {
+    pname = "purescm";
+    version = "1.0.0";
+
+    src = ./.;
+
+    dontNpmBuild = true;
+
+    npmDeps = fetchNpmDeps {
+      inherit src;
+      hash = "sha256-rSm40Q9A5xfMQIsH6QQaCcemU9LnLmvGdLmII4nApsI=";
+      postPatch = ''
+        cp ${./package-lock.json} package-lock.json
+      '';
+    };
+
+    installPhase = ''
+      mkdir -p $out/share/${pname}
+      cp -r node_modules/ $out/share/${pname}
+      mkdir -p $out/bin
+      ln -s $out/share/${pname}/node_modules/.bin/purescm $out/bin
+
+    '';
+
+    passthru.tests = if lib.versionOlder "1.0.1" version then { } else {
```

Comment:
Why this condition here?

Reply:
Tbh I wouldn't recommend anyone use anything close to the 1.0 since it's so broken, so I think we could just have 1.8 under the purescm name.

If that requires us to stop publishing under the `next` tag and use the default one instead then we can do that.

Reply:
Either is fine for me but if we keep the current version, I think we should drop the dead code.

Reply:
Great! We have [just published 1.8.2](https://www.npmjs.com/package/purescm/v/1.8.2) on the `latest` line, and we'll stop using the `next` tag - I'd like to see it dropped from this PR as well